index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>osdc.github.io</title>
+    <link href="https://fonts.googleapis.com/css?family=Assistant:200|Monoton" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <div class="main">
+      <div class="gradientbox">
+          <div class="heading">OSDC</div>
+      </div>
+      <div class="subheading">The Open Source Community of JIIT</div>
+    </div>
+    <script type="text/javascript" src="scripts/script.js"></script>
```

Comment:
This file seems to be empty. Included accidentally or yet to be updated ?

Reply:
It is yet to be updated.

Reply:
Please remove the empty file and it's inclusion from here. No point of keeping it now.

Reply:
Removed script.js